src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -320,6 +347,15 @@ public override void InitializeJobExtension(IExecutionContext executionContext)
                 executionContext.Output(StringUtil.Loc("RMUserChoseToSkipArtifactDownload"));
                 Trace.Info("Skipping artifact download based on the setting specified.");
             }
+            else
+            {
+                ReleaseArtifacts = GetReleaseArtifacts(executionContext);
+
+                if (!ReleaseArtifacts.Any())
+                {
+                    Trace.Info("No artifacts found to be downloaded by agent.");
```

Comment:
log this also in task logs (and so load message from resource)?

Reply:
This is just for info, there can be artifacts which will be downloaded using task. Printing this message will just confuse the user.